src/win/win_serial_passthrough.c

```diff
@@ -161,7 +162,15 @@ static int
 open_pseudo_terminal(serial_passthrough_t *dev)
 {
     char ascii_pipe_name[1024] = { 0 };
-    snprintf(ascii_pipe_name, sizeof(ascii_pipe_name), "\\\\.\\pipe\\86Box\\%s", vm_name);
+    if (dev->named_pipe[0] != 0) strncpy(ascii_pipe_name, dev->named_pipe, 1023);
+    else {
+        ui_msgbox(MBX_WARNING, L"Serial Passthrough: Pipe name is empty.");
+        return 0;
+    }
+    if (strnicmp("\\\\.\\pipe\\", ascii_pipe_name, sizeof("\\\\.\\pipe\\") - 1)) {
+        ui_msgbox(MBX_WARNING, L"Serial Passthrough: Pipe name is invalid.");
+        return 0;
+    }
```

Comment:
IMHO this condition is redundant as long as we properly handle `CreateNamedPipe` errors.